Bottom: 71b857381a0b3d4fc09c4f32cd8d14ca9cd6613e
Top:    b0d6229a58ba2f0195f417c6af7a0456e0d966d5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 15:21:53 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
